programs/budget/src/lib.rs

```diff
@@ -1,22 +1,5 @@
 mod budget_processor;
 
 use crate::budget_processor::process_instruction;
```

Comment:
can you make this entire file into a one-liner?

solana_sdk::process_instruction_entrypoint!(budget_processor) ?

Reply:
No, budget_processor is moving to budget_api